ethpm_types/source.py

```diff
@@ -27,6 +27,8 @@ class Source(BaseModel):
     installPath: Optional[str] = None
     type: Optional[str] = None
     license: Optional[str] = None
+    references: Set[str] = set()
+    # NOTE: Added outside of current spec for tracking dependent files
```

Comment:
```suggestion
    # Set of `Source` objects that depend on this object
    # TODO: Add `SourceId` type and use instead of `str`
    references: Set[str] = set()  # NOTE: Not a part of canonical EIP-2678 spec
```

Question: should `Source` have an `id: SourceId` field? (this would match the key used in `sources: Dict[str, Source]` under `PackageManifest`